packages/client/hmi-client/src/components/workflow/ops/calibrate-ciemss/tera-calibrate-ciemss-drilldown.vue

```diff
@@ -749,6 +750,9 @@ const generateAnnotation = async (setting: ChartSetting, query: string) => {
 };
 // ---
 
+const errorData = computed<Record<string, any>[]>(() =>
```

Comment:
Why do we need to move this into a computed? I am not understanding the issue

Reply:
`groundTruthData` which `errorData` is dependant on is a reactive variable. Which get's now computed and populated asynchronously by
```
			setupCsvAsset(dataset).then((csv) => {
				csvAsset.value = csv;
			});
```
Thus the ground truth data isn't available until csvAssts get's assigned and we need to reflect the change accordingly when that value is updated.